src/cpp/common/channel_arguments.cc

```diff
@@ -29,11 +29,35 @@
 #include <grpcpp/support/channel_arguments.h>
 #include <grpcpp/support/config.h>
 
+#include "src/core/lib/channel/channel_args.h"
 #include "src/core/lib/iomgr/exec_ctx.h"
 #include "src/core/lib/iomgr/socket_mutator.h"
 
 namespace grpc {
 
+static const grpc_arg_pointer_vtable* GrpcChannelArgsVTable() {
+  static const grpc_arg_pointer_vtable tbl = {
+      // copy
+      [](void* p) -> void* {
+        return p == nullptr
```

Comment:
No need to check for nullptr.  `grpc_channel_args_copy()` accepts a null argument.

Reply:
Removed.